### DISCUSSION TITLE
Audit accountant  app purpose

### DISCUSSION THREAD

**Discussion by digitalrevisor** [powerpaul17/nc_money#27] [Ideas]
Hi. This is a realy valuable app. Im working as auditor / accountant in Norway.
What I would sugest as a big improvement is:

1. Add a standard european trialbalance as default in app. Starts from Acc nr 1000 and ends on 9999.
2. Add standard european annualreport-template (In Moneyapp its called Group) - Somewhere here: https://www.oecd.org/tax/country-by-country-reporting-xml-schema-user-guide-for-tax-administrations.htm     -     Or here: https://stats.oecd.org/Index.aspx?DataSetCode=BPF1
3. Make it more intuitiv to start record a voucher / transaction: E.G. Date-Descr-amount-creditacc-debitacc
4. Make it more intuitiv to import a voucher / transactions from excel or csv file: E.G. Date-Descr-amount-creditacc-debitacc

Thanks for a exelent idea. If this will work, Ill buy devs a big cup og coffe.

**Comment by powerpaul17** [comment]
Hello and thanks for your kind words. Unfortunately I'm currently the only developer and not really an accountant.. :sweat_smile: At the moment the app is targeted mostly to people who want to track their expenses/income and such but I want to try to keep it universally applicable. But probably it will not be possible to get a solution for every usecase.. which does not mean I'll not think about requests/suggestions, of course.

That being said, I try to address your points, please correct me or provide more information so I can understand better:

1. Since I'm not an accountant, I do not know what a "trialbalance" is but I searched and it seems to be some kind of summary of all accounts, right? What do you mean by Nr 1000-9999? Currently accounts only have a name, you can prefix it with a number if you like.. otherwise we would have to think about adding a separate ID or something..
2. For this I would need more information. Is it a kind of "summary" in a specific format?
3. Do you mean a separate dialog for this? Currently you go into one account and at the top you can enter a new transaction and also set the destination account.
4. The problem here is that since accounts only have a name, it's not really possible to extract that information reliably from a CSV-File. What columns do you have in your data? Is it also an account number or so?

Another point: currently I don't use credit and debit columns, only one since it seemed more intuitive for non-accountant users. So I simply invert the value if you would put it into the other one. I thought about adding the possibility to use separate columns if there is demand for it..

**Reply by 1837456** [reply]
Regarding number 1 I think it's the same like the "DATEV Standardkontenrahmen (SKR)" we have in Germany. For example, as a freelance photographer I'm using SKR-03 and my income without tax is booked to account 8400 (Income 19% VAT) and tax separately to account 1776 (Income VAT 19%).
And yes, as you said, I prefixed my accounts with the appropriate numbers for better handling. 

https://www.datev.de/web/de/m/ueber-datev/datev-im-web/datev-von-a-z/skr-standard-kontenrahmen/ (German only)
